try:
    print(1/0)
except Exception as e:
    print(e)

print("allo")
